src/lib/modules/Relations/backoffice/components/RelationRemover/RelationRemover.js

```diff
@@ -39,7 +39,12 @@ export default class RelationRemover extends Component {
       >
         <Modal.Header>Confirm removal</Modal.Header>
         <Modal.Content>
-          Are you sure you want to delete this relation?
+          <p>Are you sure you want to delete this relation?</p>
```

Comment:
It looks to me (I did not check the code) that this cmp is used to remove **any** relation, also siblings and parent-child. Is this correct?
If yes, the message below might not valid for the other relation types.

Reply:
Additional message removed from parent-child relations (using the pid_type of the relation to filter only relation with other documents).
![Screenshot from 2021-09-15 15-25-12](https://user-images.githubusercontent.com/25476209/133441761-729c0a00-8b28-4e4b-b605-438d204aa37a.png)